webapp/.sass-lint.yml

```diff
@@ -0,0 +1,48 @@
+#########################
+## Sample Sass Lint File
+#########################
+# Linter Options
+options:
+  # Don't merge default rules
+  merge-default-rules: false
+  # Raise an error if more than 50 warnings are generated
+  max-warnings: 50
+  # Output file (lose loglevel from console output)
+  #output-file: 'buildlog/sass-linter.html'
+  #formatter: 'html'
+# Rule Configuration
+rules:
+  extends-before-mixins: 2
+  extends-before-declarations: 2
+  placeholder-in-extend: 2
+  mixins-before-declarations:
+    - 2
+    -
+      exclude:
+        - breakpoint
+        - mq
+
+  no-warn: 1
+  no-debug: 1
+  no-ids: 2
+  no-important: 2
+  hex-notation:
+    - 2
+    -
+      style: uppercase
+  indentation:
+    - 2
+    -
+      size: 2
+  property-sort-order:
```

Comment:
Add the [class-name-format](https://github.com/sasstools/sass-lint/blob/master/docs/rules/class-name-format.md) rule with proper configuration.

Reply:
Good catch, done!